addon/mixins/model-validator.js

```diff
@@ -237,6 +247,13 @@ export default Ember.Mixin.create({
     if(options.hasOwnProperty('only') && options.only.indexOf(property) === -1){ validateThis = false; }
     return validateThis;
   },
+  _getCustomValidator: function(validation){
+    var customValidator = validation.custom;
+    if (Ember.typeOf(validation.custom) === 'object' && validation.custom.hasOwnProperty('validation')) {
+      customValidator = validation.custom.validation;
+    } 
+    return typeof customValidator === 'function' ? customValidator : false;
```

Comment:
this is determining whether the customValidation function is located at `validations.{property}.custom` or `validations.{property}.custom.validation` (in the case that a custom message is defined), ensuring that the callback is in fact a function and returning the function to be invoked by the `_validateCustom` method on line 52.